auditlog/event.go

```diff
@@ -129,6 +130,11 @@ func NewEventWithMetadata(action Action, actionType ActionType, metadata map[str
 	return event, nil
 }
 
+// GetID gets the Event's ID
```

Comment:
I don't think this `Event` struct should know anything about the `ID` since the client cannot set it before sending it to WorkOS. It seems like `EventResponse` is the only struct that can know about the `ID`.

Reply:
Correct, I think I will move `EventResponse` to `event.go` and change its declaration to the following. 

```go
type EventResponse struct {
	Event
	ID              string `json:"id"`
	Object          string `json:"object"`
	LocationCity    string `json:"location_city"`
	LocationState   string `json:"location_state"`
	LocationCountry string `json:"location_country"`
	AppID           string `json:"app_id"`
}
```
`EventResponse` also now implements the `Iterable` interface.